src/adagrams.js

```diff
@@ -1,15 +1,157 @@
+
+const score_chart = {
```

Comment:
oop careful! use camel case in JavaScript:

```suggestion
const scoreChart = {
```